### ISSUE TITLE
Template error using shortcodes

### ISSUE THREAD

**Issue by kaloja** [11ty/eleventy#347]
I trying to use a universal shortcode with liquid templates but get this error: 

```
Problem writing Eleventy templates: (full stack in DEBUG output)
> Having trouble rendering liquid (and markdown) template ./content/anmalan.md (TemplateContentRenderError)
> cannot eval '"Zach' as value, line:4 (RenderError)
> cannot eval '"Zach' as value (TypeError)
Copied 2 items and Processed 0 files in 0.05 seconds
```

I try to get it working using the example in the docs find here: [https://www.11ty.io/docs/languages/liquid/#shortcodes](https://www.11ty.io/docs/languages/liquid/#shortcodes)

Using this in my md-file fails with the above error message:
```
{% user "Zach Leatherman" "zachleat" %}
```

Using this in my md-file renders without errors:
```
{% user "ZachLeatherman" "zachleat" %}
```

Looks like you can't have space in your shortcode variable.

I'm using Eleventy v0.6.0.

**Comment by zachleat** [comment]
Hmm, wow this is not good—sorry about that. (confirmed locally)

**Comment by zachleat** [comment]
Work for this is complete and this will be included with the upcoming 0.7.0 release. Thanks for the report!

**Comment by kaloja** [comment]
You’re welcome. Thanks for sharing such a great SSG.
